tracer/build/_build/Build.Steps.cs

```diff
@@ -358,14 +358,17 @@ private void CopyWafForProject(Project project)
         void CopyWaf(string architecture, IEnumerable<string> frameworks, AbsolutePath absolutePath, string wafFileName, string extension)
         {
             var source = LibDdwafDirectory / "runtimes" / architecture / "native" / $"{wafFileName}.{extension}";
-            var nativeDir = DDTracerHomeDirectory / architecture / $"Datadog.Trace.ClrProfiler.Native.{extension}";
             foreach (var fmk in frameworks)
             {
-                var dest = absolutePath / "bin" / BuildConfiguration / fmk / architecture;
-                CopyFileToDirectory(source, dest, FileExistsPolicy.Overwrite);
+                var targetDir = absolutePath / "bin" / BuildConfiguration / fmk;
+                var archTargetDir = targetDir / architecture;
+                CopyFileToDirectory(source, archTargetDir, FileExistsPolicy.Overwrite);
                 if (!IsWin)
                 {
-                    CopyFileToDirectory(nativeDir, absolutePath / "bin" / BuildConfiguration / fmk, FileExistsPolicy.Overwrite);
+                    var tracerNativeLib = DDTracerHomeDirectory / architecture / $"Datadog.Tracer.Native.{extension}";
+                    // The tests use the "wrong" name for the library currently
```

Comment:
NIT - I'm sure it won't be an issue for you, but if it were me I know that not adding a clear and identical TODO in all places will need to fix will make me forgot some places later.